std/typecons.d

```diff
@@ -6736,6 +6736,31 @@ struct Typedef(T, T init = T.init, string cookie=null)
             TD im() {return TD(Typedef_payload.im);}
         }
     }
+
+    /**
+     * Params:
+     *     writer = A `char` accepting
+     *     $(REF_ALTTEXT output range, isOutputRange, std, range, primitives)
+     *     fmt = A $(REF FormatSpec, std,format) which is used to represent
+     *     the value
+     * Returns:
+     *     A `string` if `writer` and `fmt` are not set; `void` otherwise.
+     */
+    void toString(W)(ref W writer, const ref FormatSpec!char fmt)
+    if (isOutputRange!(W, char))
+    {
+        formatValue(writer, Typedef_payload, fmt);
+    }
+
+    /// ditto
+    string toString()
```

Comment:
This overload should be documented and considered the "primary" one, with the other just marked as "ditto". Explain that this forwards to the wrapped value and the results between `<base type value>.to!string()` and `<Typedef'd type value>.to!string()` will be identical.

Reply:
I do not know how to describe this beautifully, just added an example.